### PR TITLE
Allow non zero exit status when thresholds not met

### PR DESCRIPTION
**Pull request by tedhtchang**
Allow runHowitzer.sh  to have non zero exit status when thresholds not met
Handle connect error before testing
Add Dockerfile example
Add k8s example
Signed-off-by: ted chang <htchang@us.ibm.com>

### CODE REVIEW

**Comment by tedhtchang** [comment]
/cc @aluu317
Should be able to try it out with:
```sh
CONFIG_FILE=./perf_test/configs/inference-test-thresholds.json TEMPLATE_DIR=./k6_test/ EXIT_ON_FAILURE=true ./perf_test/runHowitzer.sh
```

File: perf_test/k8s/howitzer_k6_test-k8s.yaml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        "description": "Example to launch groups of perf tests."
+      },
+      "test_configs": [
+        {
+          "name": "small_vus_sklearn_test",
+          "description": "2 sample sklearn inference tests, small vus.",
+          "template": "script_grpc_skmnist.js",
+          "payload": "1by64payloadSklearn.json",
+          "base_url": "modelmesh-serving:8033",
+          "model_name": "example-mnist-predictor",
```
Comment (chinhuang007) [inline perf_test/k8s/howitzer_k6_test-k8s.yaml:95]: Can we also include the model/predictor setup?

**Comment by aluu317** [comment]
@tedhtchang This is great! Thank you. Do you think it's helpful to also have a Makefile to include a `build` target, also `run.deploy` where it runs the deployment of configmap, predictor and job etc. , and `run.perf-test` targets to then call the howitzer script?

**Comment by tedhtchang** [comment]
@aluu317 Sounds like good idea. Let’s make another PR for the Makefile.

**Review by chinhuang007 — APPROVED** [review]
/lgtm

**Comment by aluu317** [comment]
yes /lgtm !

**Pull request merged**